core/test_network/src/lib.rs

```diff
@@ -378,9 +375,9 @@ where
             },
             logger: LoggerConfiguration {
                 #[cfg(profile = "bench")]
-                max_log_level: LevelEnv::WARN,
+                max_log_level: Level::ERROR.into(),
                 #[cfg(not(profile = "bench"))]
-                max_log_level: LevelEnv::INFO,
+                max_log_level: Level::TRACE.into(),
```

Comment:
I think it should stay at INFO level, and if during local debugging we need a TRACE level we can enable it temporarily. But in general especially for readability in CI, it is better to have INFO here.